local_sales/app/views/categories/_form.html.erb

```diff
@@ -1,16 +1,17 @@
-<%= form_with(model: category, local: true) do |form| %>
+<div class="lg:mt-8 mt-6">
+  <%= form_with(model: category, local: true) do |form| %>
 
-  <div>
-    <label for="fname">Categoria:</label><br>
-    <%= form.text_field :name %><br>
+  <div class="mb-4 mt-6">
+    <%= form.label :Categoria, :class => 'block text-blue-600 mb-2'%>
```

Comment:
you must change that like this <%= form.label :cover, label="Nombre", :class => 'block text-blue-600 mb-2' %>
because you would put the string correctly with accents if it need it